features/joinroom/impl/src/main/kotlin/io/element/android/features/joinroom/impl/JoinRoomView.kt

```diff
@@ -385,13 +387,14 @@ private fun JoinRoomContent(
                         Column(horizontalAlignment = Alignment.CenterHorizontally) {
                             val inviteSender = (contentState.joinAuthorisationStatus as? JoinAuthorisationStatus.IsInvited)?.inviteSender
                             if (inviteSender != null) {
-                                InviteSenderView(inviteSender = inviteSender)
+                                InviteSenderView(inviteSender = inviteSender, hideAvatarImage = shouldHideAvatars)
```

Comment:
Instead of `shouldHideAvatars`, why not using `hideAvatarImage`? `Should` has to be used when the behavior is optional.

Reply:
`Should` first meaning is about obligation, not optionality, so this makes sense. But I can change if you really want.

Reply:
Are you sure? For obligation, we use `Must`. But OK to keep it like that, not a big deal.